style.css

```diff
@@ -1,632 +1,1330 @@
+    padding: 0;
+    height: 48px;
+    width: 684px;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="text"] {
+    height: 48px;
+    width: 684px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .filter::placeholder {
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #6b778c;
+    padding-left: 16px;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="text"]:hover {
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="text"]:focus {
+    outline: none;
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"] {
+    height: 48px;
+    width: 684px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"]:hover {
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset input[type="email"]:focus {
+    outline: none;
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset .txtarea_mob {
+    display: none;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset .txtarea_dsk {
+    height: 114px;
+    width: 684px;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #091e42;
+    font-family: "Inter", sans-serif;
+    padding-top: 12px;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset textarea:hover {
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .div_field fieldset textarea:focus {
+    outline: none;
+    border: 1px solid #60c095;
+  }
+
+  .section-contact-form .div_form_container .contact-form .divbtn_field {
+    margin-top: 68px;
+  }
+
+  .section-contact-form .div_form_container .contact-form .divbtn_field .btn-form {
+    padding: 12px;
+    margin-top: 12px;
+    gap: 10px;
+    width: 129px;
+    height: 48px;
+    bottom: 8px;
+    border-radius: 4px;
+    color: #fff;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    border: 0;
+    cursor: pointer;
+  }
+
+  .hr_dsk {
+    width: 186px;
+    margin-left: 0;
+    margin-top: 170px;
+  }
```

Comment:
- [ ] Kindly make sure the line touches on both ends of you page in order to meet the provided Figma design as requested by the previous reviewers.

   - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/208168650-1fd2bdfe-3abe-4ee1-bdba-d6f889330477.png)

    - ### FIGMA DESIGN
    
   ![image](https://user-images.githubusercontent.com/87197412/208169179-fcb0222c-2755-4bc3-acfe-99d2d9c90d70.png)

Reply:
Dear @levy002.
Kindly, could you check if this is correct now?
![image](https://user-images.githubusercontent.com/65084923/208173361-07852378-f683-46c8-b391-ab2ace07e008.png)
Thank you very much!